app/controllers/users.js

```diff
@@ -0,0 +1,14 @@
+const usersService = require('../services/users');
+const logger = require('../logger');
+
+const signUp = async (req, res, next) => {
+  try {
+    const user = await usersService.createUser(req.body);
+    logger.info(`username: ${user.email}`);
+    res.status(201).send(user);
+  } catch {
+    next();
```

Comment:
This `next()` should have the error that is catched as an argument, should be something like this:
```
catch (error) {
  next(error);
}